paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -498,8 +498,9 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     mkldnn::memory::data_type src_dt =
         paddle::framework::ToMKLDNNDataType(input->type());
 
-    std::string key = platform::CreateKey(
-        src_tz, src_dt, ctx.InputName("Input") + ctx.InputName("Filter"));
+    std::string key =
+        platform::CreateKey(dev_ctx, src_tz, src_dt,
+                            ctx.InputName("Input") + ctx.InputName("Filter"));
```

Comment:
It would be good to remove the `+` operation on strings here.

Reply:
There should be no difference in the result, but prformance with `,` instead of `+` will be better.